debezium-core/pom.xml

```diff
@@ -80,6 +80,27 @@
             <artifactId>kafka-connect-avro-converter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
My point was not about Avro but rather about the following three deps (mysql-binlog-connector-java etc.)